### PR TITLE
Set the max message log for syslog to 1024, also is configurable.

### PR DESCRIPTION
**Pull request by sbchisholm**

### CODE REVIEW

File: src/Logging.jl [hunk 1]
```diff
@@ -48,9 +48,10 @@ type SysLog
     facility::LogFacility
     machine::AbstractString
     user::AbstractString
+    maxsize::UInt16
 
-    SysLog(host::AbstractString, port::Int) = new(UDPSocket(), getaddrinfo(host), (@compat UInt16(port)), LOG_USER, gethostname(), Base.source_path()==nothing ? "" : basename(Base.source_path()))
-    SysLog(host::AbstractString, port::Int, facility::LogFacility, machine::AbstractString, user::AbstractString) = new(UDPSocket(), getaddrinfo(host), (@compat UInt16(port)), facility, machine, user)
+    SysLog(host::AbstractString, port::Int) = new(UDPSocket(), getaddrinfo(host), (@compat UInt16(port)), LOG_USER, gethostname(), Base.source_path()==nothing ? "" : basename(Base.source_path()), (@compat UInt16(1024)))
```
Comment (karlrl) [inline src/Logging.jl]: Why not use `maxsize` here as well?

File: src/Logging.jl [hunk 2]
```diff
@@ -77,7 +78,7 @@ const _root = Logger("root", WARNING, STDERR)
 Logger(name::AbstractString;args...) = configure(Logger(name, WARNING, [STDERR], _root); args...)
 Logger() = Logger("logger")
 
-write_log(syslog::SysLog, color::Symbol, msg::AbstractString) = send(syslog.socket, syslog.ip, syslog.port, msg)
+write_log(syslog::SysLog, color::Symbol, msg::AbstractString) = send(syslog.socket, syslog.ip, syslog.port, length(msg) > syslog.maxsize ? msg[1:syslog.maxsize] : msg)
```
Comment (karlrl) [inline src/Logging.jl]: A bit confusing to call the variable `maxsize` and then check the string's `length()`. Consider using either `maxlength` as the name of the config var or `sizeof()` when checking to see if the message is too long.

File: src/Logging.jl (hunk 1 above)
Comment (sbchisholm) [inline src/Logging.jl]: These lines are just getting too long aren't they.  maxsize isn't available here it will just use the "default" of 1024.

Comment (karlrl) [inline src/Logging.jl]: Oh, I see. Makes sense.

**Comment by karlrl** [comment]
LGTM

**Pull request merged**